src/Sulu/Bundle/AdminBundle/Resources/js/components/Overlay/Overlay.js

```diff
@@ -153,17 +156,19 @@ class Overlay extends React.Component<Props> {
                                         />
                                     </header>
                                     <article className={overlayStyles.article}>{children}</article>
-                                    <footer className={overlayStyles.footer}>
-                                        <Actions actions={actions} />
-                                        <Button
-                                            disabled={confirmDisabled}
-                                            loading={confirmLoading}
-                                            onClick={onConfirm}
-                                            skin="primary"
-                                        >
-                                            {confirmText}
-                                        </Button>
-                                    </footer>
+                                    {footerVisible &&
```

Comment:
I think we can use `!onConfirm` and `actions.length == 0` we show no footer. As we already need for this usecase make the `onConfirm` nullable/undefined above.